src/nix-env/nix-env.cc

```diff
@@ -957,6 +953,8 @@ static void queryJSON(Globals & globals, std::vector<DrvInfo> & elems, bool prin
 
 static void opQuery(Globals & globals, Strings opFlags, Strings opArgs)
 {
+    auto & store(*globals.state->store);
```

Comment:
Nit, but @thufschmitt and I have generally been doing the C++11 style
```suggestion
    auto & store { *globals.state->store };
```
which is supposed to be less ambiguous.